testing/tests/DevExpress.ui.widgets/responsiveBox.markup.tests.js

```diff
@@ -480,18 +480,15 @@ QUnit.module("templates", () => {
     });
 
     QUnit.test("custom item renderer", function(assert) {
-        let templateContext,
-            $responsiveBox = $("#responsiveBox").dxResponsiveBox({
-                rows: [{}],
-                cols: [{}],
-                itemTemplate: function() {
-                    templateContext = this.NAME;
-                    return $("<div>test</div>");
-                },
-                items: [{ location: { row: 0, col: 0 } }]
-            });
+        const $responsiveBox = $("#responsiveBox").dxResponsiveBox({
+            rows: [{}],
+            cols: [{}],
+            itemTemplate: function() {
+                return $("<div>test</div>");
+            },
+            items: [{ location: { row: 0, col: 0 } }]
+        });
 
-        assert.equal(templateContext, "dxResponsiveBox", "Correct context");
```

Comment:
I think here is Breaking Change

Reply:
We discussed this point a week ago and decided that it is not a BC.
- we don't have mentions of this point in docs;
- we don't have tests except for current;
- this conception is not correct by ideology